### PR TITLE
SONAR-10164 Redesign the "default" badge to match other badges

### PR DESCRIPTION
**Pull request by pascal-mugnier-sonarsource**

### CODE REVIEW

File: server/sonar-web/src/main/js/app/styles/components/badges.css [hunk 1]
```diff
@@ -19,14 +19,18 @@
...
   display: inline-block;
+  vertical-align: middle;
+  height: 20px;
+  line-height: 19px;
+  border: 1px solid var(--blue);
+  box-sizing: border-box;
+  padding: 0 8px;
+  border-radius: 2px;
+  font-size: var(--smallFontSize);
+  font-weight: 400;
```
Comment (stas-vilchik) [inline server/sonar-web/src/main/js/app/styles/components/badges.css]: make sure this is needed

File: server/sonar-web/src/main/js/app/styles/components/badges.css [hunk 2]
```diff
@@ -19,14 +19,18 @@
  */
 .badge {
   display: inline-block;
+  vertical-align: middle;
+  height: 20px;
+  line-height: 19px;
+  border: 1px solid var(--blue);
+  box-sizing: border-box;
+  padding: 0 8px;
```
Comment (stas-vilchik) [inline server/sonar-web/src/main/js/app/styles/components/badges.css]: replace `8px` with `var(--gridSize)`

File: server/sonar-web/src/main/js/app/styles/components/badges.css [hunk 3]
```diff
@@ -19,14 +19,18 @@
  */
 .badge {
   display: inline-block;
+  vertical-align: middle;
+  height: 20px;
```
Comment (stas-vilchik) [inline server/sonar-web/src/main/js/app/styles/components/badges.css]: use `var(--smallControlHeight)`

File: server/sonar-web/src/main/js/app/styles/components/badges.css [hunk 4]
```diff
@@ -19,14 +19,18 @@
...
+  line-height: 19px;
```
Comment (stas-vilchik) [inline server/sonar-web/src/main/js/app/styles/components/badges.css]: `calc(var(--smallControlHeight) - 1px)`

File: server/sonar-web/src/main/js/app/styles/components/badges.css [hunk 5]
```diff
@@ -20,14 +20,11 @@
 .badge {
   display: inline-block;
   vertical-align: middle;
-  height: 20px;
-  line-height: 19px;
-  border: 1px solid var(--blue);
-  box-sizing: border-box;
-  padding: 0 8px;
+  min-height: var(--smallControlHeight);
```
Comment (stas-vilchik) [inline server/sonar-web/src/main/js/app/styles/components/badges.css]: please use `height` and fix usages

**Comment by pascal-mugnier-sonarsource** [comment]
@stas-vilchik I fixed the first two artifacts but I cannot reproduce the thrid one on my side.
image

**Comment by pascal-mugnier-sonarsource** [comment]
Badge on web api page on lines with long text has been put on the right side despite the discussion we had about it, as it seems pretty weird to me to have it on the left side:

image

**Comment by stas-vilchik** [comment]
@pascal-mugnier-sonarsource there is a problem with hover styling:

image

**Pull request merged**